reference/paths/transactions__get-transaction-actions.yaml

```
@@ -0,0 +1,30 @@
+---
+operationId: get-transaction-actions
+
+summary: Get transaction actions
+description: Gets actions for a given transaction.
+
+tags:
+  - Transactions
+
+responses:
+  "200":
+    description: Returns associated actions.
+    content:
+      application/json:
+        schema:
+          $ref: "../openapi.yaml#/components/schemas/Refund"
```

There is a typo here
```suggestion
          $ref: "../openapi.yaml#/components/schemas/Actions"
```

---

fixed in https://github.com/gr4vy/gr4vy-openapi/pull/168/commits/daa23077e261178fa4f07c0c432a587dd9b80a4a